curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6153897c27f6334716ee5abe.md

```diff
@@ -7,7 +7,12 @@ dashedName: step-10
 
 # --description--
 
-Align your `.header` text in the center, give it a padding of `32px` on all sides, and set the background to `#E0DDDD`.
+Align your `.header` text in the center. Make the text uppercase using the `text-transform` property with `uppercase` as the value.
+
+Give it a padding of `32px` on all sides. Set the background to `#0a0a23` and the text to `#fff` as the color values.
+
+Add a `border-bottom` with `4px solid #fdb347` as the value.
+
```

Comment:
It is also to avoid adding any new steps. I think the requirements are well defined and broken up into logical blocks. They are all changes to the header, I don't think it is unreasonable to have them in the same step.

Unless it is seen as a big issue I would like to keep it as is.

The uppercase change I explained in a different comment. I'd like to keep that change.